crates/compiler/builtins/bitcode/src/expect.zig

```diff
@@ -21,3 +21,33 @@ pub fn setSharedBuffer(ptr: [*]u8, length: usize) callconv(.C) usize {
 pub fn expectFailedStart() callconv(.C) [*]u8 {
     return SHARED_BUFFER.ptr;
 }
+
+extern fn shm_open(name: *const i8, oflag: c_int, mode: c_uint) c_int;
+extern fn mmap(addr: ?*anyopaque, length: c_uint, prot: c_int, flags: c_int, fd: c_int, offset: c_uint) *anyopaque;
+extern fn kill(pid: c_int, sig: c_int) c_int;
+extern fn getppid() c_int;
```

Comment:
so, we make these all `roc_` functions? also they will need support in the surgical linker (that's a formality, but still needs to be done)

Reply:
I believe that's what we decided, yep! Although in the future we may need to change them to be something more cross-platform, since Windows may have different APIs. But I don't think we should block on that right now.

Reply:
right so I wanted to use environment variables to get the name of the file that underlies the `mmap` to zig. We know windows has something similar because `memmap2` is cross-platform. But I could not get environment variables to work in our zig builtins.